test/compflags/bradc/help/userhelp.good

```diff
@@ -22,6 +22,7 @@ Optimization Control Options:
       --[no-]dead-code-elimination    Enable [disable] dead code elimination
       --fast                          Use fast default settings
       --[no-]fast-followers           Enable [disable] fast followers
+      --[no-]force-initializers       Enable [disable] default initializers
```

Comment:
I'd keep it developer only and in deprecated/ optimistically, and then consider making it a user-facing flag if we're not where we want to be closer to the release. But that's up to you.

Either way, I'd put *something* in the deprecated directory. I don't think we're obligated to remove everything from test/deprecated each release. If you don't like that, maybe make an issue or something to track removing the flag?